capplets/windows/mate-window-properties.c

```diff
@@ -315,68 +325,61 @@ main (int argc, char **argv)
         return 1;
     }
 
-    marco_settings = g_settings_new (MARCO_SCHEMA);
-
     builder = gtk_builder_new ();
     if (gtk_builder_add_from_resource (builder, "/org/mate/mcc/windows/window-properties.ui", &error) == 0) {
         g_warning ("Could not load UI: %s", error->message);
         g_error_free (error);
-        g_object_unref (marco_settings);
         g_object_unref (builder);
         return -1;
     }
 
     gtk_builder_add_callback_symbols (builder,
                                       "on_dialog_win_response",                        G_CALLBACK (response_cb),
-                                      "on_autoraise_delay_slider_value_changed",       G_CALLBACK (autoraise_delay_value_changed_callback),
                                       "on_double_click_titlebar_optionmenu_changed",   G_CALLBACK (double_click_titlebar_changed_callback),
+                                      "on_autoraise_delay_spinbutton_value_changed",     G_CALLBACK (autoraise_delay_spinbutton_value_callback),
                                       "on_titlebar_layout_optionmenu_changed",         G_CALLBACK (titlebar_layout_changed_callback),
                                       "on_focus_mode_checkbutton_toggled",             G_CALLBACK (mouse_focus_toggled_callback),
                                       "on_focus_mode_mouse_checkbutton_toggled",       G_CALLBACK (mouse_focus_toggled_callback),
                                       NULL);
 
     gtk_builder_connect_signals (builder, NULL);
 
+    #define get_widget(x) GTK_WIDGET(gtk_builder_get_object(builder, x))
```

Comment:
use all capital letters for the macro

Reply:
I actually saw this in your [PR](https://github.com/mate-desktop/mate-calc/pull/163), but its fine I will change it :)